internal/charmstore/store.go

```diff
@@ -1086,6 +1095,12 @@ func (s StoreDatabase) BaseEntities() *mgo.Collection {
 	return s.C("base_entities")
 }
 
+// Resources returns the mongo collection where resources are stored.
+func (s StoreDatabase) Resources() *mgo.Collection {
+	// TODO(ericsnow) Re-use the Entities collection?
```

Comment:
maybe we can remove this todo?  I think we decided we're not reusing entities, because resources aren't really the same thing, right?